src/components/breadcrumb/Breadcrumb.js

```diff
@@ -84,5 +84,5 @@ const Breadcrumb = ({
     );
 };
 
-export { Breadcrumb as BreadcrumbCore };
+export { Breadcrumb as BreadcrumbBase };
```

Comment:
Why this change?

Reply:
Just a cleanup. `ComponentBase` is used frequently throughout the codebase, whereas `ComponentCore` is only used here. I found it when I was searching for another component to reference